rampwf/utils/tests/test_testing.py

```diff
@@ -0,0 +1,22 @@
+import shutil
+from tempfile import mkdtemp
+from rampwf.kits import fetch_ramp_kit
+from rampwf.utils import assert_submission
+
+# later on we need to make a util to list kits to avoid repeating the same
+# list everywhere
+RAMP_KITS = ('boston_housing',
```

Comment:
I would import directly the list of available RAMP kits from `rampwf.kits.ramp_kits.py` to avoid duplication.
Not sure if its a good idea though.